src/App.js

```diff
@@ -4,7 +4,9 @@ import Dashboard from './pages/Dashboard'
 import Form from './components/Form'
 
 export default function App() {
-  const [reviews, setReviews] = useState([])
+  const [reviews, setReviews] = useState(
+    JSON.parse(localStorage.getItem('savedReview')) || []
+  )
```

Comment:
Alternatively you could use the `sessionStorage.getItem`/`sessionStorage.setItem` for keeping the data only available in this session and not for all tabs of the browser.